src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,339 @@
+@import "../../patternfly-utilities";
+
+.pf-c-sequence {
+  --pf-c-sequence--Height: #{pf-size-prem(100px)}; // fixed height hides scrollbar
+
+  // Sequence Control
+  --pf-c-sequence__control--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderBottomWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderTopColor: transparent;
+  --pf-c-sequence__control--BorderBottomColor: transparent;
+  --pf-c-sequence__control--hover--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--hover--BorderBottomWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--hover--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--hover--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--active--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--active--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--active--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--active--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--focus--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--focus--BorderBottomWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--focus--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--focus--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // Sequence Control Modifier Start
+  --pf-c-sequence__control-m-start--PositionLeft: 0;
```

Comment:
This can be removed as well as a couple others once you update the changes below.

Reply:
yep removed